doc/classes/DynamicFontData.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="DynamicFontData" inherits="Resource" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Used with [DynamicFont] to describe the location of a font file.
 	</brief_description>
 	<description>
+		Used with [DynamicFont] to describe the location of a [code].ttf[/code] or [code].otf[/code] font file for dynamic rendering during runtime.
```

Comment:
Same as with DynamicFont, it not necessary to state that just those two formats are supported.